articles/quickstart/spa/_includes/_token_renewal_server_setup.md

```diff
@@ -45,9 +51,6 @@ console.log('Listening on http://localhost:${serverPort}');
 <body></body>
 </html>
 ```
-
-The `silent.html` file exists to receive and parse the result of a token renewal. An instance of `WebAuth` from auth0.js is created and its `parseHash` method is used to produce a usable object out of the authentication result hash. This is then posted back to the parent window where the client side session can be set again.
-
 ::: note
-Be sure to add `http://localhost:${serverPort}/silent` to the **Callback URLs** section in your application's client settings.
+Add `http://localhost:${serverPort}/silent` to the **Callback URLs** section in your application's client settings.
```

Comment:
`to the **Callback URLs** field in your [client's settings](${manage_url}/#/clients/${account.clientId}/settings).`